src/cli/repo/s3_test.go

```diff
@@ -31,6 +31,7 @@ func (suite *S3Suite) TestAddS3Commands() {
 	}{
 		{"init s3", initCommand, expectUse, s3InitCmd().Short, initS3Cmd},
 		{"connect s3", connectCommand, expectUse, s3ConnectCmd().Short, connectS3Cmd},
+		// {"update s3", updateCommand, expectUse, s3UpdateCmd().Short, updateS3Cmd},
```

Comment:
nit: could you please delete this line since it's not needed anymore? Helps keep things a tad cleaner
```suggestion
```